### ISSUE TITLE
tensorflow v2.2 cuda 10.1 Training error

### ISSUE THREAD

**Issue by moonplay85** [matterport/Mask_RCNN#2284]
I am upgrading to TensorFlow 2.2 CUDA 10.1 and changing the code. But there is one problem. 
There is no problem with the existing mask detection. However, there is a problem in training. 
The h5 model is created but not recognized. Is it a version problem? 
For reference, I did a training test in TensorFlow v1.14 and it worked.
If you have the same version, please advise.

**Comment by JasonFinisher123** [comment]
how do you change the Keras.engine for tf.keras.engine? tf.keras.engine has a erorr?

**Comment by burhr2** [comment]
Kindly check this issue it may help https://github.com/matterport/Mask_RCNN/issues/2312
